pkg/testmachinery/metadata/metadata.go

```diff
@@ -72,4 +79,10 @@ func FromTestrun(tr *tmv1beta1.Testrun) *Metadata {
 			ExecutionGroup: tr.Annotations[common.LabelTestrunExecutionGroup],
 		},
 	}
+	if tr.Annotations[common.AnnotationAllowPrivilegedContainers] != "" {
```

Comment:
nit: I think this is just more convinient. 
Use `metav1.HasAnnotation()` or 
```
if value, ok := tr.Annotations[common.AnnotationAllowPrivilegedContainers]; ok {
 strconv.ParseBool(value)
}

```

Reply:
went for the second suggestion (for the first I wouldn't have known where to get the correctly typed ObjectMeta from :) )